### ISSUE TITLE
Make  server_frontend, server_backend  and server_homepage configurable when using docker

### ISSUE THREAD

**Issue by AGRG-IPA-070** [genular/simon-backend#1]
### Steps to reproduce
1. Use the docker image genular/simon:0.1.0

### Expected behaviour
Accessibility through IP or Domain should be configurable

### Actual behaviour
Accessing the frontend via another URL than localhost is not possible

### Proposed Solution
Use env vars here:
https://github.com/genular/simon-backend/blob/master/documentation/docker_images/Dockerfile

**Comment by LogIN-** [comment]
@AGRG-IPA-070 thanks for reporting this issue!

Actually configuration you are looking to achieve is currently possible, but you need to build your own docker image, and not use pre-builded one that is not intended for Server type usage.

You can find basic installation guide on "[How-To] Installation of SIMON on remote server using docker" here on [following link](https://community.genular.org/viewtopic.php?f=4&p=4#p4)

In this way you can change all configuration variables by editing **configuration.json**
